engine/src/main/java/org/hibernate/search/engine/search/predicate/dsl/CommonQueryStringPredicateFieldStep.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.predicate.dsl;
+
+/**
+ * The initial step in an "abstract" "query string" predicate definition, where the target field can be set.
+ *
+ * @param <N> The type of the next step.
+ */
+public interface CommonQueryStringPredicateFieldStep<N extends CommonQueryStringPredicateFieldMoreStep<?, ?>> {
+
+	/**
+	 * Target the given field in the simple query string predicate.
```

Comment:
Same here, javadoc needs some love if you really want to share these interfaces.